umade-client/src/pages/Authentication/ForgotPassword.jsx

```diff
@@ -0,0 +1,162 @@
+import {
+	Button,
+	CssBaseline,
+	TextField,
+	Box,
+	Typography,
+	Container,
+} from '@mui/material'
+import { useEffect, useState } from 'react'
+import { toast } from 'react-toastify'
+import '@fontsource/roboto/300.css'
+import '@fontsource/roboto/400.css'
+import '@fontsource/roboto/500.css'
+import '@fontsource/roboto/700.css'
+import Paper from '@mui/material/Paper'
+import SendIcon from '@mui/icons-material/Send'
+import axios from 'axios'
+import { redirect, useHistory } from 'react-router-dom'
+import { useNavigate } from 'react-router-dom'
+function Copyright(props) {
+	return (
+		<Typography
+			variant="body2"
+			color="text.secondary"
+			align="center"
+			{...props}
+		>
+			{'Copyright © by Umade ' + new Date().getFullYear() + '.'}
+		</Typography>
+	)
+}
+
+function ForgotPassword() {
+	const navigate = useNavigate();
+	const emailRegex = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/
+	const [email, setEmail] = useState('')
+	const [emailErrors, setEmailErrors] = useState('')
+	const [isError, setIsError] = useState(false)
+	const paperStyle = {
+		padding: 20,
+		height: '550px',
+		width: 1000,
+		margin: '50px auto',
+	}
+	useEffect(() => {
+		setEmail(email)
+	})
+	const validateInput = (field, value) => {
+		let isError = false
+		let message = ''
+		if (field === 'email' && !emailRegex.test(value)) {
+			isError = true
+			message = 'Email không hợp lệ'
+		}
+		setIsError(isError)
+		setEmailErrors(message)
+	}
+	const handleChange = (event) => {
+		setEmail(event.target.value)
+		validateInput(event.target.name, event.target.value)
+	}
+	const validateOnSubmit = () => {
+		let messages = emailErrors
+		let isErrors = isError
+
+		if (!email) {
+			messages = 'Không bỏ trống ô này'
+			isErrors = true
+		}
+
+		return { messages, isErrors }
+	}
+	const handleSubmit = async (event) => {
+		event.preventDefault()
+		const { messages, isErrors } = validateOnSubmit()
+
+		if (Object.values(messages).some((v) => v !== '' || v)) {
+			setIsError(isErrors)
+			setEmailErrors(messages)
+			return
+		} else {
+			const res = await axios.post('api/forgotPassword', {
+				receiveEmail: email,
+			})
+			console.log(res.data)
+			if (res.data.status === 0) {
+				toast.success('Mã OTP đã được gửi thành công. Chờ 3s chuyển đến trang xác thực')
+				setTimeout (()=>{
+					navigate("/otpForm")
+				}, 1000)
```

Comment:
Set timeout for 1000 unit mean 1s
```suggestion
				toast.success('Mã OTP đã được gửi thành công. Chờ 3s chuyển đến trang xác thực')
				setTimeout (()=>{
					navigate("/otpForm")
				}, 3000)
```